index.js

```diff
@@ -87,28 +87,43 @@ function listSort() {
         : separateArray
 
     var sortArray = sortUsing(separateArray.filter(x => x!=""));
-    var orderedArray = sortOrder(sortArray)
+    var orderedArray = sortOrder(sortArray);
     var jointArray = orderedArray.join(displayBy);
-
-    sortedList.innerHTML = jointArray;
+    // console.log(jointArray);
```

Comment:
It's better to completely delete code if you're no longer using it, rather than commenting it out